Changelog.md

```diff
@@ -18,6 +18,7 @@
 - Ensure starter files are passed to autotester in sorted order (#6771)
 - Resolved issue 6677 by taking timed assessment's duration into account when determining when grading can begin (#6845)
 - Fix loading results page when group is created after the due date (#6863)
+- Allow inactive groups in the graders table to be toggled for display (#6778)
```

Comment:
There's been another release! Please do another pull from upstream master and put this entry under an `[unreleased]` section.